src/signals/incident-management/containers/Dashboard/components/BarChart/vega-config-bar-chart.ts

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import type { VisualizationSpec } from 'vega-embed'
+
+export const vegaConfigBarChart: VisualizationSpec = {
+  $schema: 'https://vega.github.io/schema/vega-lite/v5.json',
+  description: 'A bar chart showing showing number of incidents per status',
+  data: {
+    values: [
+      { status: 'Heropend', nrOfIncidentsPerStatus: 0 },
+      { status: 'Extern: afgehandeld', nrOfIncidentsPerStatus: 54 },
+      { status: 'Ingepland', nrOfIncidentsPerStatus: 75 },
+      { status: 'Verzoek tot heropenen', nrOfIncidentsPerStatus: 22 },
+      { status: 'Reactie ontvangen', nrOfIncidentsPerStatus: 560 },
+      { status: 'In behandeling', nrOfIncidentsPerStatus: 82 },
+      { status: 'Gesplitst', nrOfIncidentsPerStatus: 32 },
+      { status: 'Reactie gevraagd', nrOfIncidentsPerStatus: 60 },
+      { status: 'Gemeld', nrOfIncidentsPerStatus: 1103 },
+      { status: 'Afwachting van behandeling', nrOfIncidentsPerStatus: 409 },
+      { status: 'Extern: verzoek tot afhandeling', nrOfIncidentsPerStatus: 64 },
+    ],
+  },
+  transform: [
+    {
+      calculate: 'max(2, datum.nrOfIncidentsPerStatus)',
+      as: 'nrOfIncidentsPerStatusWithMax',
+    },
+  ],
+  spacing: 5,
+  facet: {
+    field: 'status',
+    type: 'nominal',
+    title: null,
+    header: {
+      labelAnchor: 'start',
+      labelFontSize: 14,
+      labelFontWeight: 400,
+      labelPadding: 4,
+      labelFont: 'Amsterdam Sans',
+    },
+    sort: [false],
```

Comment:
sort: null is the correct way. 
<img width="829" alt="Screenshot 2023-02-07 at 11 55 05" src="https://user-images.githubusercontent.com/46756331/217225800-516404ff-f0ff-4b03-86b1-1bccedcc6bd0.png">

Reply:
If the value is null, my IDE gives an error. With false the order is maintained, but there is no error. 
![image](https://user-images.githubusercontent.com/25867841/217247187-d640ef38-719c-4d98-a941-349344ca9950.png)

Reply:
@SireeKoolenWijkstra you pass an **array** with the value **null**, and that is not allowed. If you just pass the value **null**

Reply:
Right, so that's the problem. Thanks. I changed it.